inlong-sort/sort-connectors/postgres-cdc/pom.xml

```diff
@@ -46,6 +46,11 @@
             <artifactId>sort-connector-base</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
The dependency is exists in the pom.xml of connector-base.

Reply:
same reason